### PR TITLE
Test timeline messages have correct display name with lazy loading

### PR DESCRIPTION
**Pull request by bwindels**
This needed a bit of infrastructure to create more users without spinning up more browsers, as you need at least 3 users to test the use case.
The test does the follow:
- bob creates a room
- charly-1 ... charly-10 join the room and send some messages
- alice enables lazy loading
- alice joins the room
- alice scrolls to the top of the timeline
- alice's timeline should contain the messages the charly-x users sent with the correct display name.

More lazy loading tests are coming based on this work.

### CODE REVIEW

**Comment by bwindels** [comment]
Part of https://github.com/vector-im/riot-web/issues/7287

File: src/rest/room.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const uuidv4 = require('uuid/v4');
+
+/* no pun intented */
```
Comment (dbkr) [inline src/rest/room.js]: Points awarded anyway

File: src/scenario.js [hunk 2]
```diff
@@ -21,26 +21,47 @@ const join = require('./tests/join');
...
+    await aLazyLoadingTest(alice, bob, charlies);
+}
+
+function range(start, amount, step = 1) {
+    const r = [];
+    for (let i = 0; i < amount; ++i) {
+        r.push(start + (i * step));
+    }
+    return r;
+}
```
Comment (dbkr) [inline src/scenario.js]: ftr lodash also has this, but maybe not worth pulling it in for the sake of a 5 line function

Comment (bwindels) [reply src/scenario.js]: :+1: next time a need a lodash method I'll import it

**Comment by bwindels** [comment]
Part of https://github.com/vector-im/riot-web/issues/7287

**Pull request merged**